guides/source/api_app.md

```diff
@@ -332,6 +332,28 @@ will be:
 { :person => { :firstName => "Yehuda", :lastName => "Katz" } }
 ```
 
+### Using Session Middlewares
+The following middlewares are (by default) included for session management.  If one of your API clients is a browser, you might want to add one of these back in:
+- `ActionDispatch::Session::CacheStore`
+- `ActionDispatch::Session::CookieStore`
+- `ActionDispatch::Session::MemCacheStore`
+
+The trick to adding these back in is that, by default, they are passed `session_options`
+when added (including the session key), so you can't just add a `session_store.rb` initializer, add
+`use ActionDispatch::Session::CookieStore` and have sessions functioning as normal.
+
+To be clear: sessions may work, but your session options will be ignored (i.e the session key will default
+to `_session_id`).  Instead of the initializer, you'll have to set the relevant options somewhere
+before your middleware is built (like `application.rb`) and pass them to your prefered middleware,
+like this:
+
+**application.rb:**
+```ruby
+config.session_store :cookie_store, key: '_interslice_session'
+config.middleware.use ActionDispatch::Cookies # Required for all session management
+config.middleware.use ActionDispatch::Session::CookieStore, config.session_options
```

Comment:
As far as I can tell, we should be referencing `config.action_controller.session_options` and `config.action_controller.session_store` as mentioned [here](https://github.com/rails/rails/blob/d41ec8180d3df391e5fc0189f499d8689ca27e03/guides/source/configuring.md#configuring-middleware) but that doesn't seem to set `session_options` properly. 😕

Reply:
Interesting... yeah I'm not sure why those don't work... I wonder if those are writers only... ? (that doesn't make a lot of sense...), and to be honest I can't remember how I found these values - I assume I just found them in the default middleware stack implementation: https://github.com/rails/rails/blob/master/railties%2Flib%2Frails%2Fapplication%2Fdefault_middleware_stack.rb#L62

It does seem to make sense to change them to match the docs, but if that doesn't work... then I guess it doesn't! 😉